### PR TITLE
Add forcePostProcess option, which will be cascaded from force option

### PR DESCRIPTION
**Pull request by hellosnow**

### CODE REVIEW

File: src/docfx/Models/BuildCommandOptions.cs [hunk 1]
```diff
@@ -60,6 +60,9 @@ internal class BuildCommandOptions : ICanPrintHelpMessage, ILoggable
         [Option('f', "force", HelpText = "Force re-build all the documentation")]
         public bool ForceRebuild { get; set; }
 
+        [Option("forcePostProcess", HelpText = "Force to re-process the documentation in post processors. It will be cascaded from force option.")]
+        public bool ForcePostProcess { get; set; }
```
Comment (herohua) [inline src/docfx/Models/BuildCommandOptions.cs]: >Process [](start = 29, length = 7)

processing?

File: src/docfx/SubCommands/DocumentBuilderWrapper.cs [hunk 2]
```diff
@@ -190,9 +190,12 @@ private static IEnumerable<Assembly> LoadPluginAssemblies(string pluginDirectory
 
         private static IEnumerable<DocumentBuildParameters> ConfigToParameter(BuildJsonConfig config, TemplateManager templateManager, ChangeList changeList, string baseDirectory, string outputDirectory, string templateDir)
         {
-            var parameters = new DocumentBuildParameters();
-            parameters.OutputBaseDir = outputDirectory;
-            parameters.ForceRebuild = config.Force ?? false;
+            var parameters = new DocumentBuildParameters
```
Comment (herohua) [inline src/docfx/SubCommands/DocumentBuilderWrapper.cs]: >DocumentBuildParameters [](start = 33, length = 23)

can you confirm adding ForcePostProcess here won't impact incremental build (compile + link)?

File: src/docfx/Models/BuildCommandOptions.cs (hunk 1 above)
Comment (hellosnow) [inline src/docfx/Models/BuildCommandOptions.cs]: I think 'process' here is a verb? like 'force to post process', how do you think?

---
In reply to: [94204150](https://github.com/dotnet/docfx/pull/1091#discussion_r94204150) [](ancestors = 94204150)

File: src/docfx/SubCommands/DocumentBuilderWrapper.cs (hunk 2 above)
Comment (hellosnow) [inline src/docfx/SubCommands/DocumentBuilderWrapper.cs]: Confirmed with @ansyral, already add IncrementalIgnore attribute here so will not impact incremental build(compile + link)

---
In reply to: [94204176](https://github.com/dotnet/docfx/pull/1091#discussion_r94204176) [](ancestors = 94204176)

File: src/docfx/Models/BuildCommandOptions.cs (hunk 1 above)
Comment (herohua) [inline src/docfx/Models/BuildCommandOptions.cs]: I see, it's okay

---
In reply to: [94204870](https://github.com/dotnet/docfx/pull/1091#discussion_r94204870) [](ancestors = 94204870,94204150)

File: src/docfx/SubCommands/DocumentBuilderWrapper.cs (hunk 2 above)
Comment (herohua) [inline src/docfx/SubCommands/DocumentBuilderWrapper.cs]: nice, thanks

---
In reply to: [94204965](https://github.com/dotnet/docfx/pull/1091#discussion_r94204965) [](ancestors = 94204965,94204176)

**Comment by herohua** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

**Pull request merged**